modules/ROOT/pages/osv2-guide.adoc

```diff
@@ -3,141 +3,120 @@ ifndef::env-site,env-github[]
-. Create a simple flow such as the following:
+. In Studio, create a simple flow such as the following:
 +
+--
+.The screenshot shows (*1*) the *Listener* and *Store* operations in the *os-Simple-Flow* flow.
+image::os-studio7-flow.png[Listener and Store operations in Studio]
+
 [%headers,cols="25a,75a"]
 |===
-|Building Block |*Parameters*
-|HTTP connector |Click the green plus sign +
-*Host* = `All interfaces [0.0.0.0] (Default)` +
-*Port* = `8081` +
-Click *OK* +
-*Path* = `/store` +
-*Allowed Methods* = `POST`
-|Object to String |No settings (use defaults)
-|JSON to Object | Return Class = `java.util.Map`
-|Object Store connector |Click the green plus sign. +
-Set the *Partition* field. +
-Set the *Object Store Reference* field to `_defaultUserObjectStore`. This is mandatory for
-data to be stored in Object Store v2. +
-Click *OK*. +
-Set *Operation* = *Store* +
-*Key* = `&#x0023;[payload.key]` +
-*Value Reference* = `&#x0023;[payload.value]`
-|Set Payload |*Settings* = `KEY: &#x0023;[payload.key] AND VALUE: &#x0023;[payload.value] WERE SAVED OK`
+|Connector |Parameters
+|HTTP Listener|
+*Connector configuration* = HTTP_Listener_config
+*Path* = `/store`
+|Object Store |
+*Key* = `#[payload.key]` +
+*Value* = `[payload.value]`
 |===
-+
-When complete, the flow appears:
-+
-image::osv2-studio-flow.png[]
-+
-. Save the application. Test run the application so you know it works. Right-click the canvas and click *Run project <name>*. If the application starts successfully, you can stop it. Right-click the canvas and click *Stop project <name>*.
-. Right-click your application’s name in Anypoint Studio’s Package Explorer and click *Anypoint Platform > Deploy to Cloud*.
-. At the User login window, specify your Anypoint Platform username and password, and click *Sign In*. If you don’t have an Anypoint Platform login, click *Sign Up*.
-. In Runtime Manager:
-+
-** Give the application a name. Each name is unique and becomes the URL under which your application
-is stored in Runtime Manager. The URL is in the form `<application_name>.cloudhub.io`. Ensure the application name gets a green checkmark for proper naming and being unique.
-** Set the Runtime version to Mule 3.8.5 and later, or to any Mule 4.x version.
-** Click the Use Object Store v2 checkbox.
-. After configuring your application, click Deploy Application.
-. Runtime Manager prompts you to ensure you want to make this choice.
-. Start your application.
-
-The *Application Data* label in the Runtime Manager navigation bar on the left side of the screen changes to *Object Store*.
-
+--
+. Save the application.
+. Right-click the application name in *Package Explorer* and select *Anypoint Platform > Deploy to CloudHub*.
+. If prompted, sign in to Anypoint Platform.
+. Click *Deploy Application* and click *Open in Browser*.
+. In Runtime Manager, click the *Status* column for your application.
+. In the details pane, copy the *App url* address.
 
 == Send Data to an Object Store
```

Comment:
"an object store" or "Object Store"?
GLOBALLY search for and evaluate: generic or branded product?

Reply:
an object store (but it's a title).